src/components/sections/projects/tabs/ProjectOutline.tsx

```
@@ -0,0 +1,111 @@
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import { TabsContent } from '@/components/ui/tabs';
+import type { ProjectOutline } from '@/constants/projects';
+import { Award, Lightbulb, Target, Users } from 'lucide-react';
+
+interface ProjectOutlineProps {
+  outline: ProjectOutline;
+}
+
+const ProjectOutline = ({ outline }: ProjectOutlineProps) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ProjectOutline'. Consider to delete it or rename it.

'ProjectOutline' is defined here:


(lint/suspicious/noRedeclare)